perun-openapi/openapi.yml

```diff
@@ -6866,11 +6995,126 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO findCandidates
+  /json/vosManager/findCandidates:
+    get:
+      tags:
+        - VosManager
+      operationId: findCandidates
+      summary: |
+              Find candidates for VO. Candidates can be used to create new members. Candidates are searched
+              in VOs external sources (if available). Candidates, which are already members of VO are never
+              returned even if they match searchString.
+      parameters:
+        - $ref: '#/components/parameters/id'
+        - { name: searchString, schema: { type: string}, in: query, required: true, description: 'Text to search by' }
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfCandidatesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO getCompleteCandidates
+  /json/vosManager/findCandidates/withMaxNumberResults:
+    get:
+      tags:
+        - VosManager
+      operationId: findCandidatesWithMaximumNumberResults
+      summary: |
+              Find candidates for VO with specified maximum number of results. Candidates can be used to create new members.
+              Candidates are searched in VOs external sources (if available). Candidates, which are already members of VO are never
+              returned even if they match searchString.
+      parameters:
+        - $ref: '#/components/parameters/id'
+        - { name: searchString, schema: { type: string}, in: query, required: true, description: 'Text to search by' }
+        - { name: maxNumOfResults, schema: { type: integer}, in: query, required: true, description: 'Number of maximum results' }
```

Comment:
The operations /json/vosManager/findCandidates and /json/vosManager/findCandidates/withMaxNumberResults can be merged into one with optional parameter maxNumOfResults.